cmd/thanos/bucket.go

```diff
@@ -80,6 +80,9 @@ func registerBucketVerify(m map[string]setupFunc, root *kingpin.CmdClause, name
 		Short('i').Default(verifier.IndexIssueID, verifier.OverlappedBlocksIssueID).Strings()
 	idWhitelist := cmd.Flag("id-whitelist", "Block IDs to verify (and optionally repair) only. "+
 		"If none is specified, all blocks will be verified. Repeated field").Strings()
+	deleteDelay := modelDuration(cmd.Flag("delete-delay", "Time before a block marked for deletion is deleted from bucket. "+
+		"If delete-delay is non zero, blocks are marked for deletion and compactor component is required to delete blocks from source bucket.").
```

Comment:
```suggestion
		"If delete-delay is non zero, blocks will be marked for deletion instead of being deleted immediately. Compactor component then will delete them at some point from source bucket.").
```

Reply:
done